### PR TITLE
Feat(fe)/initial i18n

### PR DESCRIPTION
**Pull request by xnyl**
This adds translation support for Bridge.

### CODE REVIEW

File: src/utils/strings.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ export const trimAddress = (address?: string, chars = 4) => {
   return `${start}...${end}`;
 };
 
-const vowels = "eaiou".split("");
+const vowels = "eaiouh".split("");
```
Comment (noiach) [inline src/utils/strings.ts:13]: If this is only used for "a" vs "an" and only for English, I think not having `h` is correct.

**Review by noiach — APPROVED** [review]
Looks good.

Do we want to leave chain and asset names as they are, or translate them as well?
And also strings in Multiwallet will need to be added at some point - not sure what the best way to do that will be.

File: src/utils/strings.ts (hunk 1 above)
Comment (xnyl) [reply src/utils/strings.ts:13]: Its more complicated, but for simplicity I will remove "h".

File: src/features/mint/components/MintStatuses.tsx [hunk 2]
```diff
@@ -365,7 +385,7 @@ export const MintDepositAcceptedStatus: FunctionComponent<MintDepositAcceptedSta
         />
       </ActionButtonWrapper>
       <SubmitErrorDialog
-        open={submittingError}
+        open={false} // TODO: crit enable: submittingError
```
[the remark this answers is not in the document]
Comment (xnyl) [reply src/features/mint/components/MintStatuses.tsx:388]: Good catch - fixed.

**Pull request merged**